### DISCUSSION TITLE
Import / Export Categories and Overide settings

### DISCUSSION THREAD

**Discussion by jbowensii** [Feu-Secret/Tokenmagic#93] [General]
I run a few servers (each with thier own license) for our gaming group.  I tried your import / export macros but it does not carry over the template settings for the individual categories or override settings.  Is there a way to do this other than put the screen side by side and cut and paste a ton?

**Comment by Feu-Secret** [comment]
This is on the todo list (check the issues). I'm aware of the problem.
